indico/modules/attachments/forms.py

```diff
@@ -101,10 +101,18 @@ class AttachmentFolderForm(IndicoForm):
     acl = AccessControlListField(_("Access control list"), [UsedIf(lambda form, field: form.protected.data)],
                                  groups=True, allow_external=True, default_text=_('Restrict access to this folder'),
                                  description=_("The list of users and groups allowed to access the folder"))
-    is_always_visible = BooleanField(_("Always Visible"), widget=SwitchWidget(),
+    is_always_visible = BooleanField(_("Always Visible"),
+                                     [HiddenUnless('is_hidden', value=False)],
+                                     widget=SwitchWidget(),
                                      description=_("By default, folders are always visible, even if a user cannot "
                                                    "access them. You can disable this behavior here, hiding the folder "
                                                    "for anyone who does not have permission to access it."))
+    is_hidden = BooleanField(_("Always Hidden in Frontend"),
```

Comment:
Hi,
I'm not that sure about the wording here. "Frontend" is not a term that a end user will necessarily understand.

Reply:
I discussed it briefly with @ThiefMaster, "Hide in event page" or similar could be an option.

Reply:
Done